src/guides/v2.4/graphql/queries/available-stores.md

```diff
@@ -39,16 +50,40 @@ query {
       {
         "id": 1,
         "code": "default",
+        "website_id": 1,
         "locale": "en_US",
         "base_currency_code": "USD",
-        "base_url": "http://example.com/"
+        "default_display_currency_code": "USD",
+        "timezone": "America/Chicago",
+        "weight_unit": "lbs",
+        "base_url": "http://example.com/",
+        "base_link_url": "http://example.com/",
+        "base_static_url": "http://example.com/pub/static/version1606976517/",
+        "base_media_url": "http://example.com/pub/media/",
+        "secure_base_url": "http://example.com/",
+        "secure_base_link_url": "http://example.com/",
+        "secure_base_static_url": "http://example.com/pub/static/version1606976517/",
+        "secure_base_media_url": "http://example.com/pub/media/",
+        "store_name": "Default Store View"
       },
       {
-        "id": 2,
+        "id": 1,
```

Comment:
Shouldn't this id be different from the previous one, since it refers to a different store?
Or maybe I'm missing the point here.

Reply:
Yup, It will be 2. sorry for the format of the response. I have update it.